src/subprocess.rs

```diff
@@ -1,26 +1,104 @@
 use crate::config::{DEFAULT_ENV_NAME, ENV_VAR_PREFIX};
 use crate::environments::Environments;
+use crate::graphql::GraphQLError;
 use crate::parameters::Parameters;
+use crate::warn_user;
 use color_eyre::eyre::Result;
 use std::collections::HashMap;
-use std::env;
+use std::fmt::{self, Display, Formatter};
+use std::str::FromStr;
+use std::{env, error};
 use subprocess::Exec;
 
 // for improved readability
 pub type EnvSettings = HashMap<String, String>;
 
+// NOTE: Hash and Debug are needed for testing... sigh
+#[derive(PartialEq, Eq, Hash, Debug)]
+pub enum Inheritance {
+    None,
```

Comment:
It looks like GitHub's code formatter thinks this `None` belongs to `Option`. Not a big deal. But, it could confuse reviewers.